### ISSUE TITLE
[CoE Starter Kit - QUESTION] QUESTION about the Creator Kit for the August upgrade

### ISSUE THREAD

**Issue by JanetK12** [microsoft/coe-starter-kit#9030]
### Does this question already exist in our backlog?

- [X] I have checked and confirm this is a new question.

### What is your question?

Hello!
I had a question about the CoE upgrade.
While doing the August upgrade, I suddenly remembered that some time ago when I raised another question in GitHUb related to not working buttons, in #8536 , the recommendation was to upgrade the CREATOR kit as well.
![image](https://github.com/user-attachments/assets/64fbcaa3-5214-4a10-a43d-c3b94c13a014)

at that time we were planning to do the July upgrade, but since the July upgrade didn't have the packages, we decided to do the August upgrade. Do we need then to do this CREATOR KIT upgrade? Or it is not required anymore?

### What solution are you experiencing the issue with?

None

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Yes you will need the latest Creator Kit to run the August release. Thanks for checkign!

**Comment by JanetK12** [comment]
Hi @Jenefer-Monroe !
Thank you for your answer!
Then could I ask 

1) where can I find the CREATOR KIT package exactly to cooperate with the August CoE Starter Kit releases, not to break anything in case right now the CREATOR KIT is adjusted for the September releases (since I see that now the September releases upgrade is out)? As we are currently doing the August upgrade.

2) do the instructions for the CREATOR KIT look the same, that we just need to "import solution" - the package of the CREATOR KIT and that's it? or there are different instructions exactly for this package?

2) As I just suddenly remembered about the CREATOR kit, it was a little bit late as I already upgraded the CoE starter kit in DEV environment with the august releases, but forgot about the CREATOR kit, so then what do I need to do? Can I just upgrade the CREATOR kit after the starter kit solutions are upgraded? 
Or do I need "to start from the beginning" -  to upgrade then the CREATOR kit and already after AGAIN upgrade the starter kit solutions with the same releases, meaning version 4.37 to version 4.37?

**Comment by Jenefer-Monroe** [comment]
Yes you can still upgrade the creator kit now in the environment. 

You can use the latest version of creator kit with either August or Sept and its found here: https://aka.ms/creatorkit 
![image](https://github.com/user-attachments/assets/05b9282e-47a3-4d68-ab18-7c429291ba8d)

Setup for creator kit is very simple, you just install it. 

Hope that helps

**Comment by Jenefer-Monroe** [comment]
correct. the order of install creator kit/core doesnt matter unless we use something brand new (not just fixed) from Creator kit. Which we did not this time.

**Comment by JanetK12** [comment]
@Jenefer-Monroe Thank you again!

and one more question:
 so I just need to import this whole zip solution in Power apps?
![image](https://github.com/user-attachments/assets/a1525ad9-a33a-43fc-a14e-8c4a689a7f75)
Do not need to extract it and take anything from there, just this whole zip package?

and from here?
![image](https://github.com/user-attachments/assets/ebaf09be-05c1-4841-ad99-a76ef791eef0)

**Comment by Jenefer-Monroe** [comment]
yes you import the solution as a whole

**Comment by JanetK12** [comment]
Hello @Jenefer-Monroe !
Just a question! Am I understanding that right that as we need to do the Strater Kit upgrade each 3 months, the CREATOR KIT does not needs to be upgraded each three months? Am I right that the CREATOR KIT needed to be installed only when doing the first set up of the Strater Kit , but after that it is not needs to be done ?

We just do the Starter Kit upgrade each 3 months; however, the CREATOR KIT we did not upgrade already for 2-3 years since the day we installed it. Only now, after the discussions with you, we did the first upgrade of the CREATOR KIT after all these years.

**Comment by Jenefer-Monroe** [comment]
Its best to keep both up to date. So I would recommend upgrading the creator kit whenever you upgrade core. 
We may not be using the new components but if we are then you wont be blocked. 
Upgrade of creator kit is very fast so it should not change your processes much. 
Hope that helps

**Comment by JanetK12** [comment]
@Jenefer-Monroe Thank you!
But I mean whether not upgrading the creator kit is, lets say, "dangerous"? because we did not upgrade it for 1-2 years, so does this somehow negatively affect the starter kit or not? would that be ok if we suddenly upgrade the starter kit after 2 years? or it can then somehow affect the starter kit solutions ?

**Comment by Jenefer-Monroe** [comment]
I dont own the creator kit so I dont know it as well as the starter kit. 

But my understanding of the way it works is that its ALM is much simpler as most of its components are PCF and so they upgrade more easily. So upgrading creator kit directly should be fine.

**Comment by Jenefer-Monroe** [comment]
There will be two cores, one with an _upgrade appended to the name. That is the one you remove. Here is some more information about the situation

## Two Solutions!
When upgrade fails the product often leaves an _Upgrade and an original solution sitting there. 
These will never converge or repair on their own, you will need to take action to resolve as a result. And until you do, you are in an unknown state.
- If you go to history you may find where the failure occurred and download details about it.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/eb0966a6-dbfb-400b-b796-5ecb5d20c1da)
- And if you don’t, then you should delete the _upgrade. And try to upgrade again to see either if it succeeds this time, or if you see the error message this time.
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/44eafb8d-a0b1-40e5-af3b-b3f1b502146c)

**Comment by Jenefer-Monroe** [comment]
Sorry I'm not following. Please share a screenshot like this so we can see what solutions you have installed in this environment
Be sure to show all records (all rows) and expand the indicated columns so that we can see full names
![image](https://github.com/microsoft/coe-starter-kit/assets/89584709/8dd512cc-21fc-4c64-adab-0f64c29b9bf2)

**Comment by JanetK12** [comment]
Hi @Jenefer-Monroe !
Here is the screenshot from the dev env., as there the upgrade went successful.
![image](https://github.com/user-attachments/assets/91a2f878-4dd3-4df9-b1cb-b68e603540d8)

is there everything alright?
but the thing is that previously when we did the upgrade, I were able to notice that in the middle of the upgrade, there needed to be two solutions - the new one and the old one, so after that the old one needed to be deleted and that means the upgrade is successful; however, last upgrades I do not see the second solution, just only one, and in the history I also cannot see the "uninstall" meaning that exactly this solution but the old version is deleted. Is that ok?

Also, as you notice from the screen, the "last upgrade" is shown as "3 months ago" even as it was one week ago. So it really feels as it is not "upgrade" but just "update" but I cam clicking "upgrade" as always, but why it is strange these times not sure.

Also, form the history, here is what I can see now - only "import"

![image](https://github.com/user-attachments/assets/42e12424-e1b6-40bd-af44-87d95182514f)


but previously it was something like this - both "import" and uninstall"

![image](https://github.com/user-attachments/assets/b59972ea-4c1a-45dc-996d-90746fdeee9b)
